_data/locales/ru.yml

```
@@ -0,0 +1,31 @@
+en:
```

```suggestion
ru:
```
root according to russian language

---

Looks good

---

90000

---

90000

---

> ```suggestion
> 
> ru:
> 
> ```
> 
> root according to russian language

---

Duplicate of #